`railway up` failing on GitHub Actions since 3.0.12

I'm running `railway up` on GitHub Actions from the root directory of my project, and supplying a valid `RAILWAY_TOKEN` environment variable.

Since the release of CLI 3.0.12 this is failing like so:

```
Run railway up
  railway up
  shell: /usr/bin/bash -e {0}
  env:
    RAILWAY_TOKEN: ***
Error: No linked project found
Error: Process completed with exit code 1.
```


Rolling back to 3.0.8 resolves the issue.

--- a/src/config.rs
+++ b/src/config.rs
@@ -148,7 +148,7 @@ impl Configs {
 
     pub fn get_closest_linked_project_directory(&self) -> Result<String> {
         if Self::get_railway_token().is_some() {
-            return Ok(self.get_current_directory()?);
+            return self.get_current_directory();
         }
         let mut current_path = std::env::current_dir()?;
         loop {

--- a/src/config.rs
+++ b/src/config.rs
@@ -147,6 +147,9 @@ impl Configs {
     }
 
     pub fn get_closest_linked_project_directory(&self) -> Result<String> {
+        if Self::get_railway_token().is_some() {
+            return Ok(self.get_current_directory()?);
+        }
         let mut current_path = std::env::current_dir()?;
         loop {
             let path = current_path

